sdk/src/azure/core/az_span_private.h

```diff
@@ -5,14 +5,25 @@
 #define _az_SPAN_PRIVATE_H
 
 #include <azure/core/az_precondition.h>
-#include <azure/core/internal/az_precondition_internal.h>
 #include <azure/core/az_result.h>
 #include <azure/core/az_span.h>
+#include <azure/core/internal/az_precondition_internal.h>
 
 #include <stdbool.h>
 
 #include <azure/core/_az_cfg_prefix.h>
 
+enum
+{
+  _az_ASCII_LOWER_DIF = 'a' - 'A',
```

Comment:
Nit,
_az_SPAN_ASCII_LOWER_DIF
_az_SPAN_MAX_SUPPORTED_FRACTIONAL_DIGITS

Reply:
I don't think these enum values are related to spans specifically so not sure if we should add that to the name. I am fine with either, but don't think there is enough reason to change it, especially since this enum isn't new in this PR, and just got moved :)

If you feel strongly that the name should indicate it is related to spans in some way, I can add that infix.

Reply:
I don't feel strongly on this.  They appear related to span and used by span.  We can fix this anytimein the future as they are internal values